cloud/helm-chart/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: zilla
+description: Zilla The Event-Driven API Gateway
+type: application
+version: 0.1.0
+appVersion: "0.9.42"
```

Comment:
This is hard coded, but should come from build version so it is always up-to-date as part of the release.

Reply:
updated, pls chk